src/serial/rx_buff.c

```diff
@@ -99,7 +99,10 @@ void rx_buff_destroy(struct rx_buff *rxb)
 bool rx_buff_read(struct rx_buff *rxb, struct Serial *s, const bool echo)
 {
         xQueueHandle h = serial_get_rx_queue(s);
-        char c = 1;
+	if (!h)
+		return false;
+
+        char c = 0xEF;
```

Comment:
Invalid UTF8 Unicode character.  Yields the symbol with the question mark in it on most terminals